Aardvark.podspec

```diff
@@ -1,11 +1,12 @@
 Pod::Spec.new do |s|
   s.name     = 'Aardvark'
-  s.version  = '3.4.0'
+  s.version  = '3.4.1'
   s.license  = 'Apache License, Version 2.0'
   s.summary  = 'Aardvark is a library that makes it dead simple to create actionable bug reports.'
   s.homepage = 'https://github.com/square/Aardvark'
   s.authors  = 'Square'
   s.source   = { :git => 'https://github.com/square/Aardvark.git', :tag => s.version }
+  s.swift_version = '4.0'
```

Comment:
Next version you will be able to designate multiple Swift versions https://github.com/CocoaPods/CocoaPods/pull/8192